libs/@wingcloud/framework/src/internal.ts

```diff
@@ -11,8 +11,8 @@ export async function compile(options: CompileOptions) {
 
   const program = ts.createProgram([options.entrypoint], {
     target: ts.ScriptTarget.ES2022,
-    module: ts.ModuleKind.Node16,
-    moduleResolution: ts.ModuleResolutionKind.Node16,
+    module: ts.ModuleKind.CommonJS,
+    moduleResolution: ts.ModuleResolutionKind.Node10,
```

Comment:
Node 10?

Reply:
Yeah, It's the same as `ModuleResolutionKind.Node` (which is deprecated to tell people that this is not the modern way to do it)

I think these options are the closest to how wing actually needs it to work